documentation/asciidoc/computers/raspberry-pi/revision-codes.adoc

```diff
@@ -342,6 +342,18 @@ NOQuuuWuFMMMCCCCPPPPTTTTTTTTRRRR
 |
 | 17: 5
 
+|
+|
+| 18: CM5
+
+|
+|
+| 19: 500
+
+|
+|
+| 1a: CM5 Lite
```

Comment:
I don't think we have any other Lite models (CM4 Lite, CM3 Lite, CM4S Lite, or CM3+ Lite...) in this chart. Can we either add them or omit CM5 Lite?

Reply:
CM5 is different - it has different numbers in OTP for EMMC and LITE. This is because it loads different device-tree files since CM5 EMMC can use the fast HS400 EMMC speed and LITE can't.
It also helps with support requests because we can easily spot the difference in logs / screenshots.